File: samples/CslaLight/cs/SimpleApp/SimpleApp/VisibilityConverter.cs
using System;

namespace SimpleApp
{
  public class VisibilityConverter : System.Windows.Data.IValueConverter
  {
    public object Convert(object value, System.Type targetType, object parameter, System.Globalization.CultureInfo culture)
    {
      if (value.ToString() == "True")
        return System.Windows.Visibility.Visible;
      else
        return System.Windows.Visibility.Collapsed;
    }

    public object ConvertBack(object value, System.Type targetType, object parameter, System.Globalization.CultureInfo culture)
    {
      return value;
    }
  }
}

File: samples/CslaLight/vb/SimpleCslaLightAppvb/SimpleApp/VisibilityConverter.vb
Public Class VisibilityConverter
  Implements System.Windows.Data.IValueConverter

  Public Function Convert(ByVal value As Object, ByVal targetType As System.Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements System.Windows.Data.IValueConverter.Convert

    If value.ToString = "True" Then
      Return System.Windows.Visibility.Visible

    Else
      Return System.Windows.Visibility.Collapsed
    End If

  End Function

  Public Function ConvertBack(ByVal value As Object, ByVal targetType As System.Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements System.Windows.Data.IValueConverter.ConvertBack
    Return value
  End Function
End Class
